packages/directory/tests/integration/components/dir-empty-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | dir-empty', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
```

Comment:
Could we use a better test name?

Reply:
"Will it blend"?

Reply:
Usually you're right, but this test description is apt in this case.